### PR TITLE
Adding Select All Functionality and Pagination Options.

### PR DESCRIPTION
**Pull request by ScottJohansen**
I think i did this right?

### CODE REVIEW

File: src/lib/paginator.svelte [hunk 1]
```diff
@@ -47,4 +60,5 @@
 			>Go &rarr;</button
 		>
 	</div>
+	<!-- <input type="number" bind:value={builder.paginator.limit}/> -->
```
Comment (ConnorVoisey) [inline src/lib/paginator.svelte]: Is this comment needed, can this line just be removed?

File: src/lib/shgrid.svelte [hunk 2]
```diff
@@ -1,4 +1,6 @@
 <script lang="ts" generics="T extends DefaultRow">
+	import { select_multiple_value } from 'svelte/internal';
```
Comment (ConnorVoisey) [inline src/lib/shgrid.svelte]: Is this import needed. Generally we should avoid importing internals unless necessary

File: src/lib/paginator.svelte (hunk 1 above)
Comment (ScottJohansen) [reply src/lib/paginator.svelte]: yes

**Review by ConnorVoisey — CHANGES_REQUESTED** [review]

**Review by ConnorVoisey — APPROVED** [review]
Good Job

**Pull request merged**